docs/cluster-management/running-locally.md

```diff
@@ -20,3 +21,98 @@ You will be prompted to enter your Client ID and Client Secret. Afterwards, type
 
 [docker]: https://www.docker.com/products/docker
 [docker-compose]: https://www.docker.com/products/docker-compose
+
+## Configuring SD-in-a-Box
+
+SD-in-a-box was intended to be an easy way to run a Screwdriver cluster locally on your development machine so you can demo its features first-hand.
+
+### Custom Docker Images
+
+Since it's powered by Docker, you can determine which images to use for it
```

Comment:
Maybe add a link to the list of available images: https://hub.docker.com/r/screwdrivercd/